### ISSUE TITLE
CommandBar violates aria-required-children

### ISSUE THREAD

**Issue by agg23** [microsoft/fluentui#14736]
### Environment Information

- **Package version(s)**: ^7.131.0
- **Browser and OS versions**: Google Chrome 84.0.4147.135, Windows 10 2004

### Describe the issue:

Standard consumption of the `CommandBar` component results in the ARIA violation: aria-required-children: Ensures elements with an ARIA role that require child roles contain them. This seems to be an extension of #12461, which mentioned failures when the `CommandBar` was empty.

### Please provide a reproduction of the issue in a codepen:

This can be trivially reproed on the example site (https://developer.microsoft.com/en-us/fluentui#/controls/web/commandbar), where every `CommandBar` reports the same issue.

#### Actual behavior:

`CommandBar` is not accessible due to failing the test.

#### Expected behavior:

`CommandBar` should be fully accessible and properly use `role` tags.

**Comment by paulgildea** [comment]
@smhigley FYI

**Comment by paulgildea** [comment]
@agg23 Thanks for the feedback!

**Comment by brandonthomas** [comment]
Also seeing this in admin-controls. Fails Accessibility Insights FastPass test. 

![image](https://user-images.githubusercontent.com/7529639/91939517-d330af00-ecaa-11ea-9758-ee8db58d3fb5.png)

**Comment by daramful** [comment]
@brandonthomas  Are you using the recent version of library? This issue seems to be fixed: https://github.com/microsoft/fluentui/pull/15511

**Comment by obvioussean** [comment]
This still reproduces if you have a command bar with all disabled items
